packages/keyring-controller/src/KeyringController.ts

```diff
@@ -237,7 +280,7 @@ export class KeyringController extends BaseController<
       // we return the account already existing at index `accountCount`
       const primaryKeyringAccounts = await primaryKeyring.getAccounts();
       return {
-        keyringState: await this.fullUpdate(),
+        keyringState: this.state,
```

Comment:
Is it important to filter `vault`, `encryptionKey` and `encryptionSalt` out of this value?

Reply:
Ops! yes, it is. I forgot this occurrence, I just changed it to `this.#getMemState()`